src/Razor/src/Microsoft.VisualStudio.RazorExtension.Dependencies/source.extension.vsixmanifest

```diff
@@ -8,7 +8,7 @@
     <!-- This is needed to mark this extension as cloud compliant. -->
     <AllowClientRole>true</AllowClientRole>
   </Metadata>
-  <Installation AllUsers="true" Experimental="true">
```

Comment:
📝 We aren't overriding a built-in extension, so this isn't considered an experimental version of an all-users extension.